.github/CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+## Release 0.6.0a3 (development release)
+
+### New features since the last release
+
+### Bug fixes
+
+### Housekeeping changes
+
+* A large number of linting corrections were made to improve the overall pylint report. These were mostly minor but essential modifications including restructuring, re-coding, optimization, updating `.pylintrc`, adding `.coveragerc`. The code quality score is improved to `A` for the released version. Check ["Files changed"](https://github.com/XanaduAI/flamingpy/pull/11/files) for details.
+* `TODO` comments have been removed exclusively from files with low code quality grades. The Dev team has created tickets to be actioned for all removed TODO comments on separate (private) FlamingPy boards.
+* `CONTRIBUTING.md`, `code_of_conduct.md`, and `CHNAGLELOG.md` were imported and modified from the StrawberryFields project. Dev team plans to extend these with customized details in future PRs.
+
+### Documentation changes
+
+### Contributors
+
+This release contains contributions from (in alphabetical order) ... See full commit details ...
```

Comment:
We would also add all contributors here; each time someone new contributes to a specific release, they would be added.
```suggestion
This release contains contributions from (in alphabetical order):

[Nariman Saadatmand](https://github.com/nariman87), [Ilan Tzitrin](https://github.com/ilan-tz)
```

Reply:
Thanks, for now, I'll prefer to either use my Name with no link to GitHub or only my handle everywhere, please. I'll remove the link for myself, happy with the rest.